app/views/shared/_social_media_accounts.html.erb

```diff
@@ -8,8 +8,8 @@
       <% socialable.social_media_accounts.each do |account| %>
         <%= content_tag_for(:li, account) do %>
           <%= link_to account.url,
-                      class: "social-media-link #{account.service_name.parameterize}" do %>
-            <span class="connect">Connect with <%= account.socialable.display_name %> on</span> <%= account.display_name %>
+                      class: "social-media-link #{account.service_name.parameterize}", lang: 'en' do %>
```

Comment:
I think we should apply the language tag to the container - the section element that opens at line 3. There would be a case for doing it this way if each one might be a different language, but since we're declaring them all to be English less repetition would be good.

Reply:
Good idea. I'll need to untranslate the heading, which might be for the best as then the entire `section` is in one language.